### DISCUSSION TITLE
Is there an official answer to #117644?

### DISCUSSION THREAD

**Discussion by rolandrgarceau** [microsoft/vscode-discussions#188] [Extension Development QnA] [answered]
I am still experiencing sluggish scrolling on an MBP similar to #117644. I have yet to attempt the suggested work around in the thread, and wanted to see if anyone may have come across a better solution than disabling gpu or shrinking window size?

**Comment by gjsjohnmurray** [comment, marked as answer]
I suggest you ask this on that issue, https://github.com/microsoft/vscode/issues/117644 unless you are an extension developer trying to deal with the problem.

Please see the second FAQ on the [pinned announcement](https://github.com/microsoft/vscode-discussions/discussions/5) to understand my response.
